sysid/src/main/native/cpp/view/DataSelector.cpp

```diff
@@ -208,34 +210,75 @@ static void AddSample(std::vector<MotorData::Run::Sample<T>>& samples,
   } else {
     float val;
     if (record.GetFloat(&val)) {
```

Comment:
Is this else branch needed anymore?

Reply:
Whoops, the whole function should have been removed.